### ISSUE TITLE
Dashboard passes some metric strings to bisect in the wrong format.

### ISSUE THREAD

**Issue by ro-berto** [catapult-project/catapult#2778]
This seems to affect page_cycler_v2

Example:

https://build.chromium.org/p/tryserver.chromium.perf/builders/win_x64_perf_bisect/builds/1460

The metric string passed by dashboard is  "metric": "pcv1-cold-timeToFirstMeaningfulPaint_avg/http___amarchoti.blogspot.com", 

Note that pcv1-cold is the InteractionRecord and timeToFirstMeaningfulPaint is the ChartName and they are joined by a dash.

The chartjson however has the results in this format.
 "pcv1-cold@@timeToFirstContentfulPaint_avg": {
      "http://amarchoti.blogspot.com": {...}}

According to the code, bisect expects the metric string to be passed in the chart/IR/trace format (https://cs.chromium.org/chromium/build/scripts/slave/recipe_modules/bisect_tester/perf_test.py?rcl=0&l=16)

Thus making bisect unable to find the metric.

**Comment by eakuefner** [comment]
The motivation for passing strings in this dash-delimited format from the dashboard was Buildbot-format output, which we don't even use any more. We should probably just start sending sanely structured data from the dashboard, like a dict that says the value name, story name, and tir_label explicitly.

**Comment by eakuefner** [comment]
A compromise that we came up with in light of the fact that test paths are ambiguous is to just pass the full test path to bisect, which has logic for dealing with 3-part test paths. Unfortunately, the CL I wrote to do this might break bisect for other tests because of the ambiguity problem. This suggests a fallback algorithm to be performed on the recipe side, which I haven't gotten around to implementing yet:

1. Treat the third component as a TIR label and look for the summary for that label in the chart JSON
2. If it's not found, treat it as a page name and use the page summary instead
3. Bail out otherwise.

I'll fix this early next week.
